Automattic/pocket-casts-ios
# Settings Sync import UserDefaults

| 📘 Part of: #1400 |
|:---:|

Imports existing saved values from `NSUserDefaults` to new `SettingsStore` type used for syncing settings.

## To test

* Install a fresh build
* Log in to an account
* Change the "Open Links In Browser" and "Row Action" settings
* Enable the `settingsSync` Feature Flag
* Kill the app
* Open the app and check that the changed settings stuck

## Checklist
